src/screens/TokenDetail.js

```diff
@@ -199,17 +227,39 @@ class TokenDetail extends React.Component {
             </div>
             {renderTokenInfo()}
           </div>
-          <div className='d-flex flex-column align-items-center config-string-wrapper mt-4 ml-lg-3 pl-lg-5 pr-lg-5'>
-            <p><strong>Configuration String</strong></p>
-            <span ref="configurationString" className="mb-2">
+          <div className='d-flex flex-column align-items-left config-string-wrapper mt-4 ml-lg-3 pl-lg-5 pr-lg-5'>
+            <p className='mb-4 mt-4'><strong>Configuration String</strong></p>
+            <div className='p-4 mb-4 flex-column align-items-center'>
+              <QRCode size={180} value={configurationString} />
+            </div>
+            <span ref="configurationString" className="mb-4 text-left">
               {getShortConfigurationString()}
               <CopyToClipboard text={configurationString} onCopy={this.copied}>
-                <i className="fa fa-clone pointer ml-1" title="Copy to clipboard"></i>
+                <i className="fa fa-lg fa-clone pointer ml-1 float-right" title="Copy to clipboard"></i>
               </CopyToClipboard>
             </span> 
-            <QRCode size={200} value={configurationString} />
-            <a className="mt-2" onClick={(e) => this.downloadQrCode(e)} download={`${this.state.token.name} (${this.state.token.symbol}) - ${configurationString}`} href="true" ref="downloadLink">Download <i className="fa fa-download ml-1" title="Download QRCode"></i></a>
+            <a className="mt-2" onClick={(e) => this.downloadQrCode(e)} download={`${this.state.token.name} (${this.state.token.symbol}) - ${configurationString}`} href="true" ref="downloadLink">
+              Download
+              <i className="fa fa-download ml-1 float-right" title="Download QRCode"></i>
+            </a>
           </div>
+          <>
+            { this.state.nft ?
```

Comment:
Isn't there a better way to write this code block?

I find ternary operators on jsx confusing to understand

Maybe create another component to render the NFT

What do you think?

Reply:
100% agreed. Just did it.